### PR TITLE
feat: change back button icon

### PR DESCRIPTION
**Pull request by imGok**
Issue : [#29](https://github.com/linagora/twake-on-matrix/issues/29)
image
image

### CODE REVIEW

File: lib/widgets/twake_components/twake_back_button.dart [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  final Color? color;
+
+  final VoidCallback? onPressed;
+
+  @override
+  Widget build(BuildContext context) {
+    assert(debugCheckHasMaterialLocalizations(context));
+    return IconButton(
+      icon: const Icon(Icons.chevron_left_rounded),
```
[the remark this answers is not in the document]
Comment (imGok) [reply lib/widgets/twake_components/twake_back_button.dart:14]: in design, corners are rounded 
image

File: lib/widgets/twake_components/twake_back_button.dart [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      iconSize: 42,
```
[the remark this answers is not in the document]
Comment (imGok) [reply lib/widgets/twake_components/twake_back_button.dart:15]: WDYM ?

[the remark this answers is not in the document]
Comment (hoangdat) [reply lib/widgets/twake_components/twake_back_button.dart:15]: I mean move it to style file

**Comment by hoangdat** [comment]
Conflicted. But no demo/screenshot = no review. Srr

**Comment by imGok** [comment]
@hoangdat just force pushed to remove c files and updated file. should be ok now (with demo)

**Pull request closed without merging**